src/pages/Locations/LocationHours/LocationHours.tsx

```diff
@@ -0,0 +1,51 @@
+import type { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import { IonList, IonItem, IonText } from '@ionic/react';
+import '../LocationItem.css';
+import type { Location } from '../../../types';
+
+interface LocationProp {
+  location: Location | undefined;
+}
+
+const LocationHours: FC<LocationProp> = ({ location }: LocationProp) => {
+  const { t } = useTranslation();
+  const amPmFormat = (hour24: string) => {
```

Comment:
1. This function should be moved out of component scope as it doesn't have any dependencies from react.